<launch>
  <arg name="veh"/>
  <arg name="package_namespace" default="calibration" />
  <arg name="node_namespace" default="buffer" />
  <arg name="input_path" doc="path to rosbag that is collected during the experiments."/>

  <!-- Set parameters -->
  <group ns="$(arg veh)/$(arg package_namespace)">
  <node pkg="$(arg package_namespace)" type="$(arg node_namespace).py" name="$(arg node_namespace)" clear_params="true" required="true" output="screen">
    <param name="veh" value="$(arg veh)"/>
    <param name="input_path" value="$(arg input_path)"/>
  </node>
  </group>

</launch>